data/campaigns/The_Hammer_of_Thursagan/scenarios/05_Invaders.cfg

```diff
@@ -207,9 +199,22 @@
             id=Angarthing
         [/recall]
 
-        [recall]
+        [role]
+            type=Dwarvish Lord,Dwarvish Dragonguard,Dwarvish Steelclad,Dwarvish Thunderguard,Dwarvish Fighter,Dwarvish Thunderer
+            [not]
+                id=Aiglondur
+            [/not]
             role=scout
-        [/recall]
+            [auto_recall][/auto_recall]
+            [else]
+                [unit]
+                    side=1
+                    type=Dwarvish Thunderer
```

Comment:
Thunderer seems an odd choice for some reason. I guess it doesn't really matter though.